public/src/bowling.js

```diff
@@ -0,0 +1,69 @@
+class Bowling {
+
+  constructor() {
+    this.MAX_FRAME_SCORE = 10;
+    this.MAX_FRAMES = 10;
+    this.firstBowl = true;
+    this.pins;
+    this.frame = 0;
+    this.frameScore;
+    this.gameScore = [[0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0]];
+  }
+  countScore(score) {
+    if(this.frame === this.MAX_FRAMES) {
+      this.firstBowl = true;
+      this.pins;
+      this.frame = 0;
+      this.frameScore;
+      this.gameScore = [[0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0]];
+    } else {
+      this._countScoreHelper(score);
+    }
+    this.pins = score;
+  }
+  _countScoreHelper(score) {
+    if(score === this.MAX_FRAME_SCORE && this.firstBowl === true) {
+      this._countHelperWhenStrike(score);
+    } else {
+      this._countHelper(score);
+    }
+  }
+  getTotalScore() {
```

Comment:
Perhaps this could this function maybe be DRY'd up slightly? Maybe by separating into to a few different functions so the 'getTotalScore' lines of codes are reduced - just an idea!